### DISCUSSION TITLE
视频简介下方新广告位

### DISCUSSION THREAD

**Discussion by regentropfen2013** [the1812/Bilibili-Evolved#1422] [功能建议 / Ideas]
![屏幕截图 2021-01-06 185207](https://user-images.githubusercontent.com/62187507/103760655-4eb1bc80-500d-11eb-807e-4652ada2497a.png)

**Comment by Geeyun-JY3** [comment]
这玩意我之前也遇到过，但目前好像还不能百分百复现 ~（广告也能灰度）~

**Reply by CKylinMC** [reply]
好像是只有满足条件才会显示，一个是没登录会显示，登陆了没有会员每隔多久或者看几个视频显示这样？

**Reply by regentropfen2013** [reply]
> 好像是只有满足条件才会显示，一个是没登录会显示，登陆了没有会员每隔多久或者看几个视频显示这样？

可我既登录了，又有会员呐😂

**Reply by CKylinMC** [reply]
啊这。。。我一直续着会员没注意。。。我只记得好像不登录会有，然后同学那边没会员有时候也有...

**Reply by the1812** [reply]
我从来没遇到过...
